Lib/os.py

```diff
@@ -651,6 +651,29 @@ def get_exec_path(env=None):
     return path_list.split(pathsep)
 
 
+def get_current_dir_name():
+    """
+    Return a string representing the current working directory taking into
+    consideration the users *PWD* environment variable if it exists. This
+    is opposed to getcwd() which dereferences symlinks in the path. This
+    function is identical to getcwd() on systems that do *not* support
+    the *PWD* environment variable.
+    """
+    cwd = getcwd()
+
+    if name == 'nt':
+        return cwd
```

Comment:
I added this so that we don't look at the `PWD` environment variable on windows. I believe this matches the behavior documented.